starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,36 @@
+const coinDeskApi = axios.create({
+    baseURL: 'http://api.coindesk.com/v1/bpi/historical/close.json'
+})
+
+function prepareData(bpi) {
+  return _.toPairs(new bpi);
+}
+
+function buildChart(dataOne) {
+  var ctx = document.getElementById("myChart").getContext('2d');
+  var config = {
+    type: 'line',
+    color: 'blue',
+    data: {
+      labels: Object.keys(dataOne),
+      datasets: Object.values(dataOne)
```

Comment:
You need to read the documentation more carefully. dataset is an array that contains an object:
```
datasets: [{
  data: Object.values(dataOne)
}]
```